### PR TITLE
Update gget to 0.28.6

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`gget`](https://bioconda.github.io/recipes/gget/README.html): **0.28.5** &rarr; **0.28.6**

[install with bioconda](http://bioconda.github.io/recipes/gget/README.html) [Conda](https://anaconda.org/bioconda/gget/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/gget`](https://github.com//bioconda/bioconda-recipes/tree/bump/gget/recipes/gget) (click to view/edit other files)
Summary | gget enables efficient querying of genomic databases
Home | [https://github.com/pachterlab/gget](https://github.com/pachterlab/gget)
Releases |[https://github.com/pachterlab/gget/tags](https://github.com/pachterlab/gget/tags)
Recipe Maintainer(s) | @nh13, @lauraluebbert
Author | `@pachterlab`
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by lauraluebbert** [comment]
Linking this here: https://github.com/bioconda/bioconda-recipes/pull/48161

TLDR: Running `conda install bioconda::gget` on some Mac OS results in the following error:
```
Output in format: Requested package -> Available versionsThe following specifications were found to be incompatible with your system:
 - feature:/osx-64::__osx==10.16=0
 - feature:|@/osx-64::__osx==10.16=0
 - gget -> matplotlib-base -> __osx[version='>=10.11|>=10.12']’
 ```

**Review by martin-g — APPROVED** [review]
